py/torch_tensorrt/fx/converters/impl/elementwise/ops.py

```diff
@@ -139,3 +139,37 @@ def rsqrt(
     )
 
     return output
+
+
+def rsub(
+    network: TRTNetwork,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    input: TRTTensor,
+    other: TRTTensor,
+    alpha: Optional[Any] = None,
+) -> TRTTensor:
+    scaled_tensor_output = other
+    if alpha is not None:
+        scaled_tensor_output = convert_binary_elementwise(
+            network,
+            target,
+            source_ir,
+            f"{name}_scaled",
+            trt.ElementWiseOperation.PROD,
+            other,
+            alpha,
+        )
+
+    output = convert_binary_elementwise(
+        network,
+        target,
+        source_ir,
+        f"{name}_output",
+        trt.ElementWiseOperation.SUB,
+        input,
+        scaled_tensor_output,
```

Comment:
according to https://github.com/pytorch/pytorch/issues/64775
rsub for (x-y) should be rsub(y, x)

Reply:
Ok, I will change this!

Reply:
Although the above issue states this, the tests are passing with same as well as multiple inputs which makes it seem like it is taking x-y as rsub(x,y). Looking into this further.